### DISCUSSION TITLE
meine Druckeinstellungen

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#69] [Ideas]
Eine Anfrage der Kolleg:innen, kann man im webGIS auf Druckeinstellungen personalisiert speichern?
![image](https://user-images.githubusercontent.com/1842500/236437609-c9ed26ac-637f-4d3b-8287-deaa0aeb1c48.png)
Inklusive der Felder:
![image](https://user-images.githubusercontent.com/1842500/236437927-d075d528-bf1b-49ed-a0be-12bac0b32c7f.png)
ähnlich wie "Meine Darstellung", "Bookmarks" etc.

Ist es denkbar, dass sowas im webGIS implementiert wird?

**Comment by gis-eni** [comment]
Irgendwie kann ich mir nicht ganz vorstellen, wie das gemeint ist. Wie sollte man das Speichen können? Du vergleichst das mit Bookmarks und Darstellungsvarianten!? Sollte man mehrere Versionen von Einstellungen und Eingaben speichern können? Bitte ein bisserl genauer definieren was du haben möchtest.

**Comment by marko-jer** [comment]
Gut! Ich habe bei den Kolleg:innen nochmals nachgefragt. Der Wunsch wäre es, die Einstellungen s.o. mit einem Projekt mitzuspeichern. Natürlich dann nur eine Version pro Projekt und dann wären die Inhalte/Darstellungen auch schon definiert/festgelegt.
Ich hoffe, dass es nun genauer ist.

**Comment by gis-eni** [comment]
Ok, jetzt kann ich mir schon eher was darunter vorstellen.
Die Einstellungen aus Werkzeugen in ein Projekt mitzuspeichern ist allerdings eine größere Baustelle und man sollte da vielleicht noch einmal nachdenken bevor am das umsetzt:

* Beschränken wir uns nur auf Drucken, oder sollen auch andere Inhalte aus Werkzeugen mitgespeichert werden.
* Wo wird diese Funktion überall angeboten? Internet, Intranet?

Bei Internet seh ich das ein bisserl Kritisch. Da werden wieder einmal zu viele Daten am Server abgelegt die vielleicht auch irgendwann Datenschutz relevant sind :/ Sollte man vielleicht vorher überlegen, was man da wirklich auf dem Server liegen hat.

**Comment by marko-jer** [comment]
Meine Gedanken dazu - im Augenblick: nur Drucken und nur für angemeldete User.
Es ist nicht dringend --> wir können nachdenken und vielleicht äußern noch andere ihre Meinung dazu.
